components/hidden_data_box/wikis/starcraft2/hidden_data_box_custom.lua

```diff
@@ -7,32 +7,39 @@
 --
 
 local Class = require('Module:Class')
+local Logic = require('Module:Logic')
 local Variables = require('Module:Variables')
+
 local BasicHiddenDataBox = require('Module:HiddenDataBox')
 local CustomHiddenDataBox = {}
 
 function CustomHiddenDataBox.run(args)
 	BasicHiddenDataBox.addCustomVariables = CustomHiddenDataBox.addCustomVariables
+	if args.team_number or Logic.readBool(args.teamevent) then
```

Comment:
it will not work for team events either since we currently only store them if they actually have a placement

Reply:
Ok, I'll just set to the false for now on SC2